src/Plugin/Field/FieldWidget/OgComplex.php

```diff
@@ -38,7 +38,7 @@ class OgComplex extends EntityReferenceAutocompleteWidget {
   public function formElement(FieldItemListInterface $items, $delta, array $element, array &$form, FormStateInterface $form_state) {
     $parent = parent::formElement($items, $delta, $element, $form, $form_state);
     // todo: fix the definition in th UI level.
-    $parent['target_id']['#selection_handler'] = 'default:og';
+    $parent['target_id']['#selection_handler'] = 'og:default';
```

Comment:
These plugin IDs weren't updated to reflect the changes when we switched from `default:og` :/ ...